calculators/helpers/ItemNetworthHelper.js

```diff
@@ -126,10 +132,11 @@ class ItemNetworthHelper {
 
     /**
      * Checks if the item is a rune
+     * @param {boolean} [allRunes=false] Whether to include all runes or just unique runes
      * @returns {boolean} Whether the item is a rune
      */
-    isRune() {
-        const isRuneId = this.itemId === 'RUNE' || this.itemId === 'UNIQUE_RUNE';
+    isRune(allRunes = false) {
```

Comment:
Generally its better to make 2 separate methods to avoid confusion when calling `this.isRune(true)` where instead it could be `this.isRune()` and `this.isUniqueRune()` or something.